packages/internal/src/build/babel/api.ts

```diff
@@ -9,7 +9,46 @@ import * as babel from '@babel/core'
 import pkgJson from '../../../package.json'
 import { getPaths } from '../../paths'
 
-import { registerBabel, RegisterHookOptions } from './common'
+import {
+  registerBabel,
+  RegisterHookOptions,
+  CORE_JS_VERSION,
+  getCommonPlugins,
+} from './common'
+
+const TARGETS_NODE = '12.16'
+// Warning! Use the minor core-js version: "corejs: '3.6'", instead of "corejs: 3",
+// because we want to include the features added in the minor version.
+// https://github.com/zloirock/core-js/blob/master/README.md#babelpreset-env
+
+export const getApiSideBabelPresets = (
+  { presetEnv } = { presetEnv: false }
+) => {
```

Comment:
This is for jest, right? Maybe we can document that decision?

Reply:
We enable preset env in every situation but building (console/exec and jest) - but will add a comment to explain why!

Reply:
✅